python/lsst/afw/math/transformBoundedField.cc

```diff
@@ -0,0 +1,86 @@
+/*
+ * LSST Data Management System
+ * Copyright 2017  AURA/LSST.
+ *
+ * This product includes software developed by the
+ * LSST Project (http://www.lsst.org/).
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the LSST License Statement and
+ * the GNU General Public License along with this program.  If not,
+ * see <https://www.lsstcorp.org/LegalNotices/>.
+ */
+
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+
+#include "numpy/arrayobject.h"
+#include "ndarray/pybind11.h"
+#include "ndarray/converter.h"
+
+#include "lsst/pex/config/python.h"  // defines LSST_DECLARE_CONTROL_FIELD
+#include "lsst/afw/table/io/python.h"
+
+#include "lsst/afw/table/io/Persistable.h"
+#include "lsst/afw/math/BoundedField.h"
+#include "lsst/afw/math/TransformBoundedField.h"
+
+namespace py = pybind11;
+using namespace py::literals;
+
+namespace lsst {
+namespace afw {
+namespace math {
+namespace {
+
+using ClsField = py::class_<TransformBoundedField, std::shared_ptr<TransformBoundedField>, BoundedField,
+                            lsst::afw::table::io::PersistableFacade<TransformBoundedField>>;
+
+PYBIND11_PLUGIN(_transformBoundedField) {
+    py::module mod("_transformBoundedField", "Python wrapper for afw _transformBoundedField library");
```

Comment:
It looks like several sub-packages escaped cleanup, including cameraGeom, fits and possibly others. I can't remove the leading underscore without fixing up the whole package and using addUnderscore=False in SConscript, which is out of scope of this ticket. So I'll remove the comment for now.